workspaces/src/types/mod.rs

```diff
@@ -65,6 +69,15 @@ impl KeyType {
             near_crypto::KeyType::SECP256K1 => Self::SECP256K1,
         }
     }
+
+    /// Length of the bytes of the public key associated with this key type.
+    #[allow(clippy::len_without_is_empty)] // This is an associated length.
+    pub const fn len(&self) -> usize {
```

Comment:
No clear public use cases based on data or anything, but just seemed nice to have when people just have the `KeyType` accessible instead of the `PublicKey` for whatever reason

Reply:
Okay, although I'm concerned `len` might be a misnomer, considering rust's naming convention. Maybe we can name it after it's equivalent in sdk? https://github.com/near/near-sdk-rs/blob/89f8c2d776c8d00feb45239fdb15725527f1894b/near-sdk/src/types/public_key.rs#L23